modules/org.kie.kogito.builder/install.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 set -e
+
 cd /workspace
+CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
```

Comment:
```suggestion
CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o rhpam-kogito-operator main.go
```
since using this in the yamls

Reply:
Could be rhpam-kogito-operator-manager as suggested by zanini

Reply:
Would also need to change the reference here: https://github.com/kiegroup/rhpam-kogito-operator/pull/1/files#diff-eda18bf907bf5619e4d297d9a06cd67ea32859f6fbff3199904d6e238091788cR42

Reply:
yes, thanks.